modules/dnn/perf/perf_net.cpp

```diff
@@ -198,6 +198,10 @@ PERF_TEST_P_(DNNTestNetwork, Inception_v2_SSD_TensorFlow)
 
 PERF_TEST_P_(DNNTestNetwork, YOLOv3)
 {
+    applyTestTag(
+            CV_TEST_TAG_VERYLONG,
```

Comment:
> `CV_TEST_TAG_VERYLONG`

Why do we need to add this to resolve out of memory issue?

Reply:
Fixed.